tools/update_ops.raku

```diff
@@ -252,16 +252,16 @@ sub MAIN($file = "src/core/oplist") {
 
     # Generate NQP Ops file.
     my $nf = open("lib/MAST/Ops.nqp", :w);
-    $nf.say("# This file is generated from $file by tools/update_ops.p6.");
+    $nf.say("# This file is generated from $file by tools/update_ops.raku.");
     $nf.say("");
     $nf.say(%op_constants<NQP>);
     $nf.close;
 
-    # Generate a p6 Ops file into the tools directory
+    # Generate a Raku Ops file into the tools directory
     my $pf = open("tools/lib/MAST/Ops.pm", :w);
-    $pf.say("# This file is generated from $file by tools/update_ops.p6.");
+    $pf.say("# This file is generated from $file by tools/update_ops.raku.");
     $pf.say("");
-    $pf.say(%op_constants<P6>);
+    $pf.say(%op_constants<Raku>);
     $pf.close;
 
```

Comment:
This change (and one below) would seem to imply a change needed in whatever it is downstream that uses the tool - which isn't in this repository.

Reply:
Oh, sign, I misread generating code vs generated code.